6001-7000/LIT6018MalkaSanbat.xml

```diff
@@ -0,0 +1,117 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6018MalkaSanbat" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">መልክአ፡ ሰንበተ፡ ክርስቲያን፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Malkǝʾa sanbata krǝstiyān</title>
+                <title xml:lang="en" corresp="#t1">Image of the Sabbath of the Christians</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="ES"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="GundaGunde144"></witness>
+                    <witness corresp="BAVet276"></witness>
+                    <witness corresp="EMML6993"></witness>
+                    <witness corresp="EAP526112"></witness>
+                </listWit>
+                
+                
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
```

Comment:
@DenisNosnitsin1970, is this a fair clarification?
```suggestion
                <ab>In spite of its title and inclusion in manuscript collections of and lists of malkəʾ-hymns, it might rather be considered a "pseudo-malkəʾ" as its stanzas do not salute body parts (as in a typical malkəʾ) but rather other, abstract features (like its greatness [stanza 9], its honour [stanza 10], and its beauty [stanza 11]).</ab>
```